game/game/playercontrol.cpp

```diff
@@ -395,6 +396,8 @@ void PlayerControl::marvinF8(uint64_t dt) {
   pl.clearSpeed();
   pl.quitIneraction();
   pl.setAnim(AnimationSolver::Idle);
+  if (pl.isSwim())
+    pl.invalidatePhysics(dp);
```

Comment:
Why invalidation is needed only in `swim` case?

Reply:
Because if `isDive()` is true `isSwim()` is true as well.

Reply:
I mean what is the point of invalidating in not all cases? Is there any issues?